### DISCUSSION TITLE
Portrait mode feature?

### DISCUSSION THREAD

**Discussion by FlufshyFluf** [keiwando/toonsquid#190] [Ideas]
I just bought Toonsquid and didn't know that its landscape only ^^"
I draw with my fingers so its hard for me to reach with my thumbs the center of the canvas ^^"
I tried drawing sideways but that didn't really work-
Would portrait mode ever be possible? I understand if not!

**Comment by keiwando** [comment]
There are a certain number of other changes and additions to the UI that I need to make to make portrait mode possible but I do want to add it eventually.

Thanks for the feedback and for the patience!

**Comment by cirquesbreeze** [comment]
Upvoted.

I believe that incorporating portrait mode into ToonSquid would be a fantastic enhancement that many animators, including myself, would greatly appreciate.

**Comment by alex-dearden** [comment]
+1 for portrait mode please
